### PR TITLE
chore: remove default assignment

### PR DESCRIPTION
**Pull request by ovflowd**
Closes https://github.com/openjs-foundation/cross-project-council/issues/1090

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/ad-hoc-meeting.md [hunk 1]
```diff
@@ -3,7 +3,6 @@ name: ad-hoc-meeting
 about: Used to organize ad-hoc meetings
 title: "[Ad hoc meeting] "
 labels: ''
-assignees: 'brianwarner'
```
Comment (ljharb) [inline .github/ISSUE_TEMPLATE/ad-hoc-meeting.md:6]: instead, can we make the CPC team be the default assignee?

**Review by eemeli — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/ad-hoc-meeting.md (hunk 1 above)
Comment (tobie) [reply .github/ISSUE_TEMPLATE/ad-hoc-meeting.md:6]: Brian was assigned for operational reasons (adding it to the agenda and maybe setting up zoom?). So no need to ping the CPC instead.

**Review by tobie — APPROVED** [review]

**Review by timmywil — APPROVED** [review]

**Review by VoltrexKeyva — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/ad-hoc-meeting.md (hunk 1 above)
Comment (ljharb) [reply .github/ISSUE_TEMPLATE/ad-hoc-meeting.md:6]: Ok, then who’s doing the operational steps now?

Comment (ovflowd) [reply .github/ISSUE_TEMPLATE/ad-hoc-meeting.md:6]: @ljharb you cannot assign teams on GitHub Issue templates.

Comment (ovflowd) [reply .github/ISSUE_TEMPLATE/ad-hoc-meeting.md:6]: We can add the agenda label if needed.

**Review by Relequestual — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/ad-hoc-meeting.md (hunk 1 above)
Comment (ljharb) [reply .github/ISSUE_TEMPLATE/ad-hoc-meeting.md:6]: perhaps @bensternthal should be the assignee?

Comment (mhdawson) [reply .github/ISSUE_TEMPLATE/ad-hoc-meeting.md:6]: +1 on @bensternthal he is the logical replacement

Comment (bensternthal) [reply .github/ISSUE_TEMPLATE/ad-hoc-meeting.md:6]: Happy to help feel free to put my name in there.

File: .github/ISSUE_TEMPLATE/ad-hoc-meeting.md [hunk 2]
```diff
@@ -3,7 +3,6 @@ name: ad-hoc-meeting
 about: Used to organize ad-hoc meetings
 title: "[Ad hoc meeting] "
 labels: ''
```
Comment (tobie) [inline .github/ISSUE_TEMPLATE/ad-hoc-meeting.md:5]:
```suggestion
labels: ''
assignees: 'bensternthal'
```

**Review by tobie — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**